app/models/user.rb

```diff
@@ -1,10 +1,14 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Even though you activated the `:confirmable` module in the migration file, you need to activate it here too. This will ensure that your application confirms email upon sign-up before having full access to the pages meant for users. Kindly activate the `:confirmable` module in this model file for users. 👍 😎

Reply:
@Gambit142 Done making the changes